microservice-transaction-sample/README.md

```diff
@@ -37,6 +37,7 @@ The endpoints defined in the services are as follows:
 
 - Customer Service
   - `getCustomerInfo`
+  - `getCustomerInfoForTwoPhaseCommit`
```

Comment:
Is there any better name?

Reply:
One possible idea could be to include an optional transactionID field in the `getCustomerInfo` request. Based on whether the `transactionID` field is set, we could then decide to use either the one-phase or two-phase interface. However, I'm not sure if this is feasible. What do you think?

Reply:
Good point! Actually, I tried it first. It should work. But I decided to create a new endpoint considering the following things:
1. Containing the logic to decide which transaction manager to use would introduce some complexity. I wasn't sure it was a good idea for those who read the sample
2. [We described](https://github.com/scalar-labs/scalardb-samples/pull/57/files#diff-f304274bfac8570556ffd80151fb0ac7ab6fa408004ced15d8dde605eb1932e9R53-R64) which endpoints are called to use each functionality. If `getCustomerInfo` works in either of normal (non-2PC interface) transaction and 2PC interface transaction, I thought it would be complicated

For 1, on second thought, the following code would be still simple:
```java
  @Override
  public void getCustomerInfo(
      GetCustomerInfoRequest request, StreamObserver<GetCustomerInfoResponse> responseObserver) {
    if (request.hasTransactionId()) {
      return getCustomerInfoInternalInTwoPhaseCommit(request, responseObserver);
    }
    else {
      return getCustomerInfoInternal(request, responseObserver);
    }
  }
```

For 2, I think we can add a good description to the section in the document like "`getCustomerInfo` works in 2PC interface when it's called by the OrderService". But I'm still not sure which is better, merging them or not 🤔 . I don't have a strong opinion on it, though.

BTW, adding the following comment might be useful for the readers of this sample project.

> Merging `getCustomerInfoForTwoPhaseCommit` endpoint into `getCustomerInfo` endpoint may be a good way to reduce the number of endpoints. We intentionally separate the endpoints to keep them simple as a sample application.

Reply:
@brfrn169 As we discussed, I'll merge `getCustomerInfoForTwoPhaseCommit` into `getCustomerInfo`.